cookiecutter.json

```diff
@@ -5,7 +5,7 @@
     "author_email": "code@inmanta.com",
     "license": "ASL 2.0",
     "copyright": "{% now 'local', '%Y' %} Inmanta",
-    "repo_url": "https://github.com/inmanta/",
-    "package_repo_url": "https://pypi.org/simple/",
+    "modules_v1_repo_url": "https://github.com/inmanta/",
```

Comment:
This rename will break any scripts that use the project template and provide an explicit repo_url. @edvgui do you know if we have any of those? I don't think we do on core's side.

Reply:
We don't for the inmanta-module-factory: https://github.com/inmanta/inmanta-module-factory/blob/0050fdf9d530d2ddd17e326493a5ebe059c8b6b1/src/inmanta_module_factory/builder.py#L263
Burt we do for the restbase module: https://code.inmanta.com/solutions/modules/restbase/-/blob/master/inmanta_plugins/restbase/generator/app.py#L109

Reply:
If the renaming is absolutely required, we could update this code (it is offline code), but if we don't have to do it, it is always better

Reply:
I would keep it as is then, unless there is a strong reason to do so, @Hugo-Inmanta?

Reply:
Ah, it was part of the ticket...

Reply:
Perhaps there is a way to keep the old option but hidden so we can gradually deprecate it?

Reply:
I don't see an easy way out for this.
- Cookiecutter supports hidden variables (usable internaly and not prompted to the user), starting with an underscore, so that would mean renaming `repo_url` to `_repo_url` so nothing is gained compared to renaming to `modules_v1_repo_url`
- If the variable is not present in the cookiecutter.json, it won't be considered when passed by the extra_context argument in scripts (see [source](https://github.com/cookiecutter/cookiecutter/blob/ae83c1d1c15605912449ed31d276350363053a07/cookiecutter/generate.py#L52)) even when added manually to the dict in a pre-script hook (e.g. with `{{ cookiecutter.update({"repo_url": "repo_url_from_prescript"}) }}`

Reply:
Ok, let's discuss this in the planning meeting tomorrow.